app/views/posts/show.html.erb

```diff
@@ -8,6 +8,11 @@
   </div>
   <div>
     <p><%=@post.text%></p>
+  </div>
+  <div class="post">
+    <% if can? :delete, @post %>
+      <%= link_to 'Delete', user_post_path(@current_user, @post), method: :delete, data: { confirm: 'Are you sure?' } %>
+    <% end %>
```

Comment:
- [ ] It will be great to ensure that the delete post action doesn't cause an error. Just like the previous reviewer has said, when you try to delete a post the following error pops up ⬇️ 
![Screenshot 2023-05-27 at 12 39 31](https://github.com/AyoMoses1/Gems_Blog/assets/32361963/3ed660fb-7735-494f-9793-afe0b223595e)

Kindly fix this issue